src/components/CopyButton.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <button type="button"
+          class="button photo_copy-btn">
+    <slot v-if="!success" default />
+    <template v-if="success">Copied!</template>
```

Comment:
This could be a `v-else` instead

Reply:
I prefer the explicitness of multiple v-ifs (they can be re-ordered, moved, etc. without depending on their order in the code) but I can switch if you want, just let me know :)

Reply:
I'd prefer the `v-if`/`v-else` if you don't mind. So to keep it like in regular JS code where it's normal to see the `if/else` pattern

Reply:
Resolved.